src/Offer/ReadModel/JSONLD/OfferLDProjector.php

```diff
@@ -148,11 +145,7 @@ public function handle(DomainMessage $domainMessage)
         }
     }
 
-    /**
-     * @return string[]
-     *   An associative array of commands and their handler methods.
-     */
-    private function getEventHandlers()
+    private function getEventHandlers(): array
```

Comment:
The `string[]` doc block here was more specific, so maybe we should keep that?
I think we can be even more specific and mark it as `array<string,string>` in the doc block.